python/src/trezorlib/cli/debug.py

```diff
@@ -19,7 +19,11 @@
 import click
 
 from .. import mapping, messages, protobuf
-from ..debuglink import TrezorClientDebugLink, record_screen
+from ..client import TrezorClient
+from ..debuglink import TrezorClientDebugLink
+from ..debuglink import prodtest_t1 as debuglink_prodtest_t1
+from ..debuglink import record_screen
```

Comment:
i'm very surprised that isort is OK with this chunk, but if yes, let's go ahead

Reply:
yep, I also double checked when I saw it :D